features/admins/admin_skins.feature

```diff
@@ -112,3 +112,26 @@ Feature: Admin manage skins
     And I should not see "#title"
     And I should not see "text-decoration: blink;"
 
+  Scenario: Admin can change the default skin
```

Comment:
This snuck into the wrong file when we resolved the merge conflict on #2632 and merged it earlier today (15 January, 2017). Putting it into the right file here.